ddtrace/ext/ci_visibility/util.py

```diff
@@ -0,0 +1,26 @@
+from functools import wraps
+
+from ddtrace.internal.logger import get_logger
+
+
+log = get_logger(__name__)
+
+
+def _catch_and_log_exceptions(func):
+    """This decorator is meant to be used around all methods of the CIVisibility classes.
+
+    It accepts an optional parameter to allow it to be used on functions and methods.
+
+    No uncaught errors should ever reach the integration-side, and potentially cause crashes.
+    """
+
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        # try:
+        return func(*args, **kwargs)
+
+    # except Exception:
```

Comment:
Seems like this whole function might be dead code?

Reply:
Ah crud. Thanks for noticing. Forgot I undid the change in [4ca35a5](https://github.com/DataDog/dd-trace-py/pull/8711/commits/4ca35a55e5851a9c6e45ed4ac11e6f7c002184cc).